tests/unit/conftest.py

```diff
@@ -180,3 +181,9 @@ def action_example() -> Action:
 @pytest.fixture
 def actions_example(action_example) -> Actions:
     return Actions.parse_obj([action_example])
+
+
+@pytest.fixture
```

Comment:
Is it worth doing

```suggestion
@pytest.fixture(autouse=True)
```

then we don't have to remember to use this? I can't imagine where we'd _want_ to sleep in tests.